.github/workflows/codeql-analysis.yml

```diff
@@ -63,5 +72,70 @@ jobs:
     #   make bootstrap
     #   make release
 
+    - name: Perform CodeQL Analysis
+      uses: github/codeql-action/analyze@v2
+
+  analyze-java:
```

Comment:
The PR title says "csharp", which one is true?

Reply:
My typo. I updated the title but seems it is not saved. The current PR is for Java.

Got some issue in C# build, so reverted it for now.